org.adempiere.base/src/org/compiere/model/MInvoiceLine.java

```diff
@@ -388,10 +388,7 @@ public void addDescription (String description)
 	 */
 	public void setM_AttributeSetInstance_ID (int M_AttributeSetInstance_ID)
 	{
-		if (M_AttributeSetInstance_ID == 0)		//	 0 is valid ID
-			set_Value("M_AttributeSetInstance_ID", Integer.valueOf(0));
-		else
-			super.setM_AttributeSetInstance_ID (M_AttributeSetInstance_ID);
+		super.setM_AttributeSetInstance_ID (M_AttributeSetInstance_ID);

```

Comment:
The super method does the same when (M_AttributeSetInstance_ID == 0), this if statement is pointless